src/views/createEditCommunity/__tests__/create.test.tsx

```diff
@@ -1220,18 +1265,23 @@ describe('create community', () => {
             })
         );
 
-        communityCreateMock.mockImplementationOnce(() =>
-            Promise.resolve({
-                data: communityDummyData,
-                error: undefined,
-            })
-        );
+        communityCreateMock.mockImplementationOnce(() => {
+            return new Promise((resolve, reject) => {
+                setTimeout(() => {
+                    resolve({
+                        data: communityDummyData,
+                        error: undefined,
+                    });
+                }, 1000);
```

Comment:
What's the reason for adding the timeout?

Reply:
This was an attempt to make the call last longer and somehow the states which relies on this (ie. submitting) changes.  didn't work and is removed from the code already.